modules/eloqua_api_auth_fallback/eloqua_api_auth_fallback.links.menu.yml

```diff
@@ -0,0 +1,5 @@
+eloqua_api_auth_fallback.settings:
+  title: 'Eloqua API Authentication Settings'
+  route_name: eloqua_api_redux.settings
+  description: 'Configure Eloqua API Authentication Settings.'
```

Comment:
can we name this to something a little more descriptive? maybe:
"Configure Eloqua API resource owner password credentials" or something reflecting this?

Reply:
Done